src/components/views/room_settings/RoomProfileSettings.tsx

```diff
@@ -270,8 +271,11 @@ export default class RoomProfileSettings extends React.Component<IProps, IState>
                             disabled={!this.state.canSetName}
                         />
                         <Field
-                            className="mx_ProfileSettings_controls_topic"
-                            id="profileTopic"
+                            className={classNames(
```

Comment:
I don't think we should use `classNames` here as it doesn't' seem to be very necessary

Reply:
It is for style lint (max length on one line).